packages/tasit-contracts/migrations/2_deploy_contracts.js

```diff
@@ -2,6 +2,9 @@ var SimpleStorageWithRemoved = artifacts.require(
   "./SimpleStorageWithRemoved.sol"
 );
 
+var FullNFT = artifacts.require("./FullNFT.sol");
+
 module.exports = function(deployer) {
   deployer.deploy(SimpleStorageWithRemoved, "Hello World!");
+  deployer.deploy(FullNFT);
```

Comment:
No strong preference as well. I've kept in the same file because these contracts have simple deploy. Your call.

Reply:
We can keep it here for now. But maybe once we hit 1.0 we should add new migrations for new contracts we add.

Reply:
For a SQL migration which this is modeled after, I think the idea is that you do a new migration each time you would be changing the prod db that might have changed state since the last migration was run

Reply:
OK, I'll create a new file.

Reply:
Once you read the message above, feel free to resolve this with no changes or issues

Reply:
Resolving this